specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md

```diff
@@ -6,6 +6,11 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-entitysearch
-  output-folder: $(node-sdks-folder)/lib/services/entitySearch/lib
+  package-version: 1.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/entitySearch
```

Comment:
@daschult - FTYI, We have rollup packages for different cognitive services category ([search](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesSearch), [vision](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesVision), [language](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesLanguage)). Somehow we will have to make sure that the rollup packages also get updated and published when we generate a package for an individual congnitive service.